src/desktop/assets/artist2.js

```diff
@@ -1 +1,38 @@
 import 'desktop/apps/artist2/client'
+import cta from 'desktop/apps/artist/client/cta.coffee'
+import metaphysics from 'lib/metaphysics.coffee'
+import Artist from 'desktop/models/artist.coffee'
+import { data as sd } from 'sharify'
+
+const query = `
+query ArtistCTAQuery($artistID: String!) {
+  artist(id: $artistID) {
+    href
+    name
+    cta_image: image {
+      thumb: resized(width: 150, version: "square") {
+        url
+      }
+    }
+    cta_artists: artists(size: 1) {
+      image {
+        thumb: resized(width: 150, version: "square") {
+          url
+        }
+      }
+      name
+    }
+  }
+}
+`
+const send = {
+  method: 'post',
+  query,
+  variables: { artistID: 'andy-warhol' },
```

Comment:
Oh yea, question- should I just try to parse/munge `window.location.href` or something here? Or is there a way to bootstrap that in (all we need is the id).

Reply:
You can import the file (from components, per comment below) and then [insert it around here](https://github.com/artsy/force/blob/master/src/desktop/apps/artist2/client.js#L10) and pass in an artistID. By the time we're here can pull it off of the window (though maybe? that value is on `sd`?)

Reply:
If we were inside of the react component tree we would be able to grab this from `this.props.router` but since we're not we'll have to pluck it out somehow

Reply:
I'll add it to sharify then, `res.locals.sd.ARTIST_ID=..` or whatever.